test/integration/framework/test_context.go

```diff
@@ -32,6 +33,8 @@ func registerFlags(t *TestContextType) {
 		fmt.Sprintf("The maximum duration of a single call.  If unset, will default to %v", DefaultSingleCallTimeout))
 	flag.BoolVar(&t.IsOpenShiftTest, "openshift-tests", false,
 		"If set, the test will only run the Openshift case. By default, it's set to false, only the non-Openshift cases get run")
+	flag.BoolVar(&t.IsIstioEnabled, "istio-enabled", false,
```

Comment:
Why are we disabling it by default?

Reply:
we only open it when we want to test istio case,  @enlinxu , please have a look at the change on `.travis.yml`. We added a line there, by default, we only test the regular cases, the namespace isn't injected. When we want to test sidecar cases, we set it to ture and specify the parameter `-ginko.focus` to filter out the cases.  In such way,  we're trying to reuse the code without adding more resize&move cases for sidecar.